lib.go

```diff
@@ -46,24 +47,42 @@ func isArray(i interface{}) bool {
 	}
 }
 
-func New(endpoint string, opts ...graphql.ClientOption) *Client {
+func New(endpoint string, secret string, opts ...graphql.ClientOption) *Client {
+
+	var tokenString string
+	if secret != "" {
+		token := jwt.New(jwt.SigningMethodHS256)
```

Comment:
Are we sure we want to use symmetric signing?

Reply:
Yes, that is how prisma service auth works at the moment.